fedimint-server/src/config.rs

```diff
@@ -33,12 +33,6 @@ use crate::net::connect::{parse_host_port, Connector};
 use crate::net::peers::NetworkConfig;
 use crate::{ReconnectPeerConnections, TlsTcpConnector};
 
-/// Default port for communication with peers when not specified (0x1FED)
-pub const DEFAULT_P2P_PORT: u16 = 8173;
```

Comment:
Even though we don't put it into the url, we might want to interpret a *missing* port + fedimint prefix as this port.